scripts/profile_rsc.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2024 SiFive, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You should have received a copy of LICENSE.Apache2 along with
+# this software. If not, you may obtain a copy at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import urllib3
+import sys
+import json
+import time
+
+def make_add_job_json(size, blob_id):
+    visible_files = [{
+        "path": "vis." + blob_id + "." + str(i),
+        "hash": "536246ae5ed7e59ee3593656144644f2e733a9d86c7e54730c44a5fe9eff246c"
+    } for i in range(size) ]
+
+    output_files = [{
+        "path": "out." + blob_id +  "." + str(i),
+        "mode": 420,
+        "blob_id": blob_id
+    } for i in range(size) ]
+
+    return {
+        "cmd": [47,117,115,114],
+        "cwd":".",
+        "env":[80,65,84,72,61,47],
```

Comment:
Are these seemingly-magic numbers documented anywhere?

Reply:
These numbers themselves are made up. Its just the byte representation of the command/environment string of the job. The fact that you need to serialize those values into a byte array is documented in the api type